eldritch/events.py

```diff
@@ -3516,6 +3517,14 @@ def resolve(self, state):
       state.event_stack.append(self.check)
       return
 
+    if self.deactivate and not isinstance(self.deactivate, Event):
```

Comment:
You could probably create the deactivate event in the constructor, given that it doesn't need any information about what happened.
That being said, I think the game logic will automatically deactivate items at the end of a combat round. You might be able to only deactivate spells here? Or maybe you want to update the [spell deactivation](https://github.com/gummyboars/lunboks/blob/master/eldritch/eldritch.py#L669) to also work on CombatRounds against EventMonsters? Maybe that would be too much. Try out just deactivating the spells here and see if that works.